### DISCUSSION TITLE
DSS VoIP Notifier

### DISCUSSION THREAD

**Discussion by Dregi56** [indomus/forum#2022] [HUB: Home Assistant]
Qualcuno ha installato questo add-on? Come si fa ad installarlo? Trovo il link su Github ma da HACS di Hassio mi dice sempre che non esiste.

**Comment by snakuzzo** [comment]
Ma se è un addon che c'entra HACS?
HACS serve per i custom component e le custom card. Gli addon sono altra roba.

Devi andare sotto Settings > Addons e aggiungere lì la URL del repository.

P.S. riguardati la differenza tra addon e custom component, perché secondo me non ti è chiarissima.

**Reply by Dregi56** [reply]
Sempre grazie per l'aiuto. 
Nella sezione repository di componenti aggiuntivi aggiungendo il link (https://github.com/sdesalve/hassio-addons ricevo questo errore: expected a URL for dictionary value @ data['repository']. Got '(https://github.com/sdesalve/hassio-addons'
